### PR TITLE
References part deux

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

**Review by paulrobertlloyd — COMMENTED** [review]
Wondering if we could use markdown definition lists for user needs (think we support these), ie:

```
As a referee
: I need to provide a useful reference
: So that I can help the candidate with their application and get back to my day as quickly as possible
```

File: app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Give a reference iteration
+description:
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md]: Post needs a description

File: app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Some referees may not want to, or be able to give a reference. This slows down the candidate's application process.
+
+**If** we let users tell us that they won't give a reference
+**Then** we'll be able to notify the candidate and get another referee quickly
+**We'll know this works when**:
+- Referees click the refuse link in the email and confirm their refusal
+
+### One big text box for the reference
+
+We don't have any evidence that referees and providers need anything more sophisticated or bespoke than a single text box. So for now, we're just keeping parity with UCAS.
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md]: Albeit with a smaller box and lower word count

File: app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+We're also wary about asking directly if the candidate is safe to work with children. Providers like this, but we feel we need to be careful how we ask this. The absence of the flag might be seen as a bad when really the referee didn't know either way.
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md]:
```suggestion
We're wary about asking explicitly if the candidate is safe to work with children. Providers liked this, but we need to be careful how we ask the question. The absence of the flag might be seen as reason alone to reject a candidate when really the referee didn't know either way.
```

**Review by fofr — CHANGES_REQUESTED** [review]
Can you include details on how this iteration improves upon http://bat-design-history.herokuapp.com/apply-for-teacher-training/give-a-reference

File: app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+We also don't have enough confidence in the 1 to 10 scales, which providers described as ‘subjective’. A single box is the lowest risk.
+
+**If** we let users enter everything into 1 big text box
+**Then** we'll be able to see what problems arise with referees and providers
```
Comment (fofr) [inline app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md:50]: Maybe: 
We'll know this works when providers successfully use the references given to make decisions

**Review by fofr — APPROVED** [review]
Nice! :100:

**Pull request merged**